korgan/miniapp_legal_workspace.py

```
@@ -0,0 +1,213 @@
+    try:
+        answer, sources = await core.service.consult(
+            question,
+            case_context=context,
+            language=payload.language,
+        )
+    except Exception as exc:
+        if business.settings.consultation_limit_enabled and used:
+            await business.release_free_consultation(quota_id)
+        raise HTTPException(
+            status_code=502,
+            detail="Не удалось выполнить Stress Test. Лимит запроса не списан — попробуйте ещё раз.",
```

Ruff (0.16.3)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)